### ISSUE TITLE
Should we add videos?

### ISSUE THREAD

**Issue by jellis18** [nanograv/pulsar_timing_school#4]
There are several pulsar timing and GW videos on youtube. Should we add them?

**Comment by stevertaylor** [comment]
Depends what they are. If they are just promo videos, like the John Rowe or the more recent NANOGrav video, then those are not so instructional. But if they go into the concepts then sure.

**Comment by stevertaylor** [comment]
Also, perhaps we should make short (~1 minute) long videos on pulsar-timing concepts. Kind of like a mini-Coursera course.

**Comment by jellis18** [comment]
@stevertaylor, yes I was thinking of maybe our CAJAGWR talks and @scottransom has a good one too. Maybe these...

https://www.youtube.com/watch?v=2f0mdVEUpO0
https://www.youtube.com/watch?v=lyNBmgZ-j4M
https://www.youtube.com/watch?v=GJh4_uHS35M
https://www.youtube.com/watch?v=gyBnBCxvQQ4
https://youtu.be/6Yb7EE-tWlc
https://youtu.be/rKpFFTGbaDc
https://www.youtube.com/watch?v=PsO5oTIgh1c

In terms of the ~1 minute video that is a good idea but could end up being a lot of work.

**Comment by stevertaylor** [comment]
Yeah, these look good. The ~1 minute videos might need some work, but it could be an interesting diversion for Friday war-rooms.

**Comment by cmingarelli** [comment]
Hey guys,
Do you have Pablo Rosado’s video there? It won a prize a few years ago, https://www.youtube.com/watch?v=SVHm3tAjnNQ <https://www.youtube.com/watch?v=SVHm3tAjnNQ>.
If you’re more in the mood for having a laugh, Kat Grover and I did this video during our PhD, https://www.youtube.com/watch?v=0ItjOF987QI <https://www.youtube.com/watch?v=0ItjOF987QI>. It now has 21k views for some odd reason… who knows?!
Cheers,
Chiara
